scripts/x-charts-pro.exports.json

```diff
@@ -204,7 +207,13 @@
   { "name": "DefaultizedSeriesType", "kind": "TypeAlias" },
   { "name": "defaultOnBeforeExport", "kind": "Function" },
   { "name": "Direction", "kind": "TypeAlias" },
+  { "name": "elGR", "kind": "Variable" },
```

Comment:
Shouldn't we avoid exporting the locales from the index file? 

When someone is doing `import { LineChartPro } from '@mui/x-charts-pro'`, I believe these locales will still need to be evaluated by the bundler, even if they're dropped in the end. @Janpot can probably confirm here. 

We aren't exporting the locales from the community package.

Reply:
Some bundlers like vite would have some overhead in dev mode with barrel files. Next.js should do the right thing. IMO, until vite fixes that issue, we should just advice performance-conscious users to not user the barrel file.

Reply:
I've reverted it to be in line with our current approach

Reply:
Left comments in the index files as well